### PR TITLE
Daos 951 test: Adding rebuild test performing io using ior mpiio

### PR DESCRIPTION
**Pull request by saurabhtandan**
Added test to perform rebuild for data which was written using ior mpiio driver.
Create Pool of 30G
Perform I/O using MPIIO driver of IOR. Use 2-way replication.
Kill a server. Exclude the Server and trigger rebuild.
Perform another set of I/O
Monitor rebuild statistics for completion of the rebuild
Verify data integrity via IOR verify feature
Run this test for different I/O sizes

### CODE REVIEW

File: src/tests/ftest/pool/rebuild_with_ior.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    This class contains tests for pool rebuild that feature I/O going on
+    during the rebuild using IOR.
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        super(RebuildWithIOR, self).setUp()
+
+        self.mpio = None
+        self.hostfile_clients = (
```
Comment (mjean308) [inline src/tests/ftest/pool/rebuild_with_ior.py]: This is not needed.  It is part of TestWithServers

File: src/tests/ftest/util/ior_utils.py [hunk 2]
```diff
@@ -131,7 +131,7 @@ def run_ior_daos(client_file, ior_flags, iteration, block_size, transfer_size,
 # pylint: disable=too-many-arguments
 def run_ior_mpiio(basepath, mpichinstall, pool_uuid, svcl, numprocs, hostfile,
                   ior_flags, iteration, transfer_size, block_size,
-                  display_output=True):
+                  display_output=True, oclass=3, test_file="testFile"):
     """
```
Comment (mjean308) [inline src/tests/ftest/util/ior_utils.py]: You may need to rebase with the latest ior utils

File: src/tests/ftest/pool/rebuild_with_ior.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        # Verify the pool information after rebuild
+        checks["pi_ndisabled"] = targets
+        self.assertTrue(
+            self.pool.check_pool_info(**checks),
+            "Invalid pool information detected after rebuild")
+        self.assertTrue(
+            self.pool.check_rebuild_status(rs_errno=0, rs_done=1),
+            "Invalid pool rebuild error number detected after rebuild")
+
+        # perform second set of io using IOR
```
Comment (skirvan44) [inline src/tests/ftest/pool/rebuild_with_ior.py]: What does this test?

Comment (saurabhtandan) [reply src/tests/ftest/pool/rebuild_with_ior.py]: Checks couple of things.
First that rebuild is complete.
Second, that rebuild completed without any issues.

**Pull request merged**